ubuntu22.04/precompiled/Dockerfile

```diff
@@ -66,6 +67,10 @@ RUN apt-get update && apt-get install -y --no-install-recommends nvidia-driver-$
 RUN apt-get update && apt-get install --download-only --no-install-recommends -y linux-objects-nvidia-${DRIVER_BRANCH}-server-${KERNEL_VERSION} \
     linux-signatures-nvidia-${KERNEL_VERSION} \
     linux-modules-nvidia-${DRIVER_BRANCH}-server-${KERNEL_VERSION}
+# add support for nvidia open source driver packages during runtime    
+RUN apt-get update && apt-get install --download-only --no-install-recommends -y linux-objects-nvidia-${DRIVER_BRANCH}-server-open-${KERNEL_VERSION} \
+    linux-signatures-nvidia-${KERNEL_VERSION} \
```

Comment:
We already install the signatures package in the previous RUN command, so we can remove.
```suggestion
```